### ISSUE TITLE
updated data

### ISSUE THREAD

**Issue by edixiong** [nelson-liu/lost-in-the-middle#14]
Hi,

I notice that qa data is re-uploaded with comment "Re-upload correct data". I wonder what changes are made?

Thanks!

**Comment by nelson-liu** [comment]
When converting our private repo to this publicly-released version, I made a mistake when retrieving hard negatives (distractors) for input queries (in short, I mistakenly used a different embedder for the query and the passage). Thus, the hard negatives in the initial data from this repo aren't as hard as the data that we used in our experiments. I re-ran the retrieval to get the correct (hardest) negatives, and that's in the new version of the data release.

I re-ran the experiments in the paper on the previously-uploaded incorrect data and found that the results were virtually unchanged, which is perhaps interesting unto itself.

Hope that helps, happy to answer any further questions you might have about it.
